lib/msal-common/src/network/RequestThumbprint.ts

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { ClientInfo } from "../auth/ClientInfo";
+import { ICrypto } from "../crypto/ICrypto";
+import { StringUtils } from "../utils/StringUtils";
+
+export class RequestThumbprint {
```

Comment:
this feels odd to have in addition to `AccessTokenKey` since they are basically the same fields--though they represent two different ideas. suggestions?